app/models/bulk_submission.rb

```diff
@@ -149,13 +149,6 @@ def process
 
             begin
               orders_processed = orders.map(&method(:prepare_order)).compact
-              library_types = orders_processed.map do |order|
-                order.request_options['library_type'] unless order.request_options.nil?
-              end.uniq
-              if library_types.size > 1
-                errors.add :spreadsheet, "Submission #{submission_name} has multiple library types: #{library_types.join(', ')}."
-                next
-              end
 
```

Comment:
We've dramatically simplified the way we find the request to fail, making this restriction uneccessary.